test/integration/render-tests/regressions/mapbox-gl-js#8817/style.json

```diff
@@ -0,0 +1,67 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "width": 64,
+      "height": 64,
+      "operations": [
+          ["pauseSource", "geojson"],
```

Comment:
Would be great to have a comment explaining why `pauseSource` is needed for this test.